skabase/SKAAlarmHandler/SKAAlarmHandler/SKAAlarmHandler.py

```diff
@@ -129,97 +107,148 @@ def delete_device(self):
 
     def read_statsNrAlerts(self):
         # PROTECTED REGION ID(SKAAlarmHandler.statsNrAlerts_read) ENABLED START #
+        """
+        Reads number of active alerts.
+        :return: Number of active alerts
```

Comment:
What docstring format is being followed here?  From the LSST DM guidelines (which is what I believe is to be used for SKA), the numpy docstring format should be used:  https://developer.lsst.io/python/numpydoc.html

@flyingfrog81 Is that correct?

Reply:
@ajoubertza the current docstrings are as per reStructured Text format. As per [SKA coding guidelines](https://developerskatelescopeorg.readthedocs.io/en/latest/development/python-codeguide.html), PEP257 conventions are to be used for docstrings. Since the current docstrings are not as per PEP257, we will update the docstrings.

Reply:
Thanks for the link.  I didn't know there was a new guideline.  Note that PEP257 doesn't specify how exactly the parameters should be documented, which may lead to inconsistency.  I think the numpy docstring format is really good - it is compatible with PEP257.  It can be used with readthedocs.

I will ask @flyingfrog81 about this via email, as there was no response here.

Reply:
@jbmorgado has been working on our python standards recently and he might contribute his own opinion on this.

Reply:
@flyingfrog81 I agree that it would be waste of effort to enforce a standard on a legacy package that is pulled in.   I recommend that all new packages, like this one, follow a _detailed_ docstring guideline.  My choice is numpydoc, but if @jbmorgado has something better in mind, that is OK too.

The best way to ensure consistency is for SKA to dictate the standard.  I think that the earlier we do that, the better for the project.